.gitignore.txt

```diff
@@ -0,0 +1,356 @@
+
```

Comment:
gitignore is always good, but should NOT have the .txt extension.  Git will not recognize it with this.